presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -139,10 +135,11 @@ public OperatorFactory duplicate()
     private final List<AccumulatorFactory> accumulatorFactories;
     private final Optional<Integer> hashChannel;
     private final int expectedGroups;
+    private final Closer closer = Closer.create();
```

Comment:
Let's not use `Closer`. You made `HashAggregationBuilder` implement `Closeable` and `throws IOException` in the `close` method so that you can use `Closer`. But it really doesn't make sense at all for them to throw `IOException`.

If we end up need something like this in a lot of places. We can implement our own closer in Presto or airlift. For now, just do it the poor man's way.